src/commands/ext-install.ts

```diff
@@ -114,17 +116,49 @@ async function installExtension(options: InstallExtensionOptions): Promise<void>
 /**
  * Command for installing a extension
  */
-export default new Command("ext:install <extensionName>")
-  .description("install an extension, given <extensionName> or <extensionName@versionNumber>")
+export default new Command("ext:install [extensionName]")
+  .description(
+    "install an extension, provide [extensionName] or [extensionName@versionNumber] or omit to see all available extensions."
+  )
   .option("--params <paramsFile>", "name of params variables file with .env format.")
   .before(requirePermissions, ["firebasemods.instances.create"])
   .before(ensureExtensionsApiEnabled)
   .action(async (extensionName: string, options: any) => {
+    const projectId = getProjectId(options, false);
+    const paramFilePath = options.params;
+    let learnMore = false;
+    if (!extensionName) {
```

Comment:
Optional: Separate the contents of the if statement out into a helper function (promptForOfficialExtension?), so that we can add a similar feature for '''firebase ext:info''' in the future

Reply:
done